### PR TITLE
context menu item localized #7

### PR DESCRIPTION
**Pull request by hamletv**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -14,7 +13,7 @@ browser.contextMenus.create({       // Context menu - right click on-demand func
   id: "discover-image-text",
   title: "Discover Image Text",     // browser.i18n.getMessage("contextMenuItemDiscoverImageText"),
   contexts: ["image"],
-  onclick: info => {
+  onclick: info => {                // info: image info. if so can we check if alt attr present?
```
Comment (yzen) [inline background.js]: This comment is unrelated to this patch, if it needs to stay, it should be in a different pull request.

File: background.js [hunk 2]
```diff
@@ -14,7 +13,7 @@ browser.contextMenus.create({       // Context menu - right click on-demand func
   id: "discover-image-text",
   title: "Discover Image Text",     // browser.i18n.getMessage("contextMenuItemDiscoverImageText"),
   contexts: ["image"],
-  onclick: info => {
+  onclick: info => {
```
Comment (yzen) [inline background.js]: nit: please remove trailing white space here.

File: content.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 const myPort = browser.runtime.connect({name:"content-port"}); // Connection to background.js
 
 myPort.onMessage.addListener((message) => {  // Listener for message from background script.
   // If image repeats in page, text will populate in all images.
   let images = document.querySelectorAll(`img[src="${message.srcUrl}"]`);
 
   for(let image of images){
-    image.setAttribute("alt", "Possible text: " + message.result.text);
+    image.setAttribute("alt", browser.i18n.getMessage("messageContent",
+    message.result.text));
```
Comment (yzen) [inline content.js]: nit: indentation needs to be fixed (add 2 spaces)

Comment (hamletv) [reply content.js]: Done. Thanks yzen.
